docs/reference/quickstart/full-text-tutorial.asciidoc

```diff
@@ -0,0 +1,428 @@
+{"index":{"_id":"5"}}
+{"title":"Crispy Oven-Fried Chicken","description":"Get that perfect crunch without the deep fryer! This oven-fried chicken recipe delivers crispy, juicy results every time. A healthier take on the classic comfort food.","author":"Maria Rodriguez","date":"2023-05-20","category":"Main Course","tags":["chicken","oven-fried","healthy"],"rating":4.9}
+----
+// TEST
+
+[discrete]
+[[full-text-tutorial-match-query]]
+=== Perform basic full-text searches
+
+[discrete]
+==== `match` query
+
+The <<query-dsl-match-query, `match`>> query is the standard query for full-text AKA "lexical" search.
+Input is analyzed before performing the search by tokenizing text, removing possessives, converting to lowercase, filtering out stop words, and applying stemming.
+
+Search the `description` field for "fluffy pancakes":
+
+[source,console]
+----
+GET /cooking_blog/_search
+{
+  "query": {
+    "match": {
+      "description": {
+        "query": "fluffy pancakes", <1>
+        "analyzer": "standard" <2>
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+<1> By default, the `match` query uses `OR` logic between the resulting tokens. This means it will match documents that contain either "fluffy" or "pancakes", or both, in the description field.
+<2> {es} defaults to the analyzer defined in the field mapping, so this field is unnecessary here. This is only required if you're using a different analyzer at search time, which is an <<different-analyzers,advanced use case>>.
+
+[discrete]
+==== Refine the `match` query
+
+Specify the `and` operator to require both terms in the `description` field.
+
+[source,console]
+----
+GET /cooking_blog/_search
+{
+  "query": {
+    "match": {
+      "description": {
+        "query": "fluffy pancakes",
+        "operator": "and"
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+
+This stricter query returns no results.
+
+Use the <<query-dsl-minimum-should-match,`minimum_should_match`>> parameter to specify the minimum number of terms a document should have to be included in the search results.
+
+Search the title field to match at least 2 of the 3 terms: "fluffy", "pancakes", or "breakfast".
+This is useful for improving relevance while allowing some flexibility.
+
+[source,console]
+----
+GET /cooking_blog/_search
+{
+  "query": {
+    "match": {
+      "title": {
+        "query": "fluffy pancakes breakfast",
+        "minimum_should_match": 2
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+
+
+[discrete]
+[[full-text-tutorial-multi-match]]
+=== Search across multiple fields at once
+
+A <<query-dsl-multi-match-query,`multi_match`>> query runs the same query against multiple fields.
```

Comment:
Worth adding field boosts here. I would personally recommend most text search users use `multi_match` over `match`.

Reply:
>  I would personally recommend most text search users use multi_match over match.

Cool, what would be the concise explanation for this recommendation?

Reply:
Handled in https://github.com/elastic/elasticsearch/pull/114353/commits/92e47af3da87e2e4948863f57677789eb3e2176b